src/OpenSage.Game/Logic/Object/GameObject.cs

```diff
@@ -250,6 +251,11 @@ public bool IsDamaged
                 _rallyPointMarker = AddDisposable(new GameObject(rpMarkerDef, gameContext, owner, parent));
             }
 
+            if(Definition.KindOf?.Get(ObjectKinds.Projectile) ?? true)
```

Comment:
```suggestion
            if (Definition.KindOf?.Get(ObjectKinds.Projectile) ?? false)
```

I think we should default to `false`, shouldn't we?